WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -3,36 +3,77 @@ package org.wordpress.android.viewmodel.pages
 import android.arch.lifecycle.LiveData
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
+import kotlinx.coroutines.experimental.CommonPool
+import kotlinx.coroutines.experimental.launch
+import org.wordpress.android.R
 import org.wordpress.android.R.string
-import org.wordpress.android.fluxc.Dispatcher
 import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.model.page.PageModel
+import org.wordpress.android.fluxc.model.page.PageStatus.PUBLISHED
+import org.wordpress.android.fluxc.store.PageStore
 import org.wordpress.android.ui.pages.PageItem
-import org.wordpress.android.ui.pages.PageItem.Action
+import org.wordpress.android.ui.pages.PageItem.Divider
 import org.wordpress.android.ui.pages.PageItem.Empty
+import org.wordpress.android.ui.pages.PageItem.ParentPage
+import org.wordpress.android.viewmodel.ResourceProvider
 import javax.inject.Inject
 
 class PageParentViewModel
-@Inject constructor(val dispatcher: Dispatcher) : ViewModel() {
-    private val mutableData: MutableLiveData<List<PageItem>> = MutableLiveData()
-    val data: LiveData<List<PageItem>> = mutableData
+@Inject constructor(private val pageStore: PageStore, private val resourceProvider: ResourceProvider) : ViewModel() {
+    private val _pages: MutableLiveData<List<PageItem>> = MutableLiveData()
+    val pages: LiveData<List<PageItem>>
+        get() = _pages
```

Comment:
do you think this approach is more readable? I think functionality wise it's the same.

Reply:
You're right. I just used `get()` because it's everywhere else. I'll update other places and get rid of it. It'll save a few lines. Thanks!